sources/packages/api/scripts/setupDB.ts

```diff
@@ -1,29 +1,23 @@
-require("../env_setup");
-import { createConnection } from "typeorm";
-const config = require("../ormconfig");
+import { ormConfig, simsDataSource } from "../src/database/config/ormconfig";
 
 /**
  * Script main execution method
  */
 (async () => {
   try {
-    // Create Connection
+    // Create DataSource
     console.log("**** Running setupDB ****");
-    delete config.entities;
-    const connection = await createConnection({
-      ...config,
-      logging: ["error", "warn", "info"],
-    });
-    await connection.query(`CREATE SCHEMA IF NOT EXISTS ${config.schema};`);
-    await connection.query(`SET search_path TO ${config.schema}, public;`);
-    await connection.query(`SET SCHEMA '${config.schema}';`);
+    const connection = await simsDataSource.initialize();
+    await connection.query(`CREATE SCHEMA IF NOT EXISTS ${ormConfig.schema};`);
+    await connection.query(`SET search_path TO ${ormConfig.schema}, public;`);
+    await connection.query(`SET SCHEMA '${ormConfig.schema}';`);
     console.log(`**** Running migration ****`);
     await connection.runMigrations();
-    await connection.close();
+    await connection.destroy();
```

Comment:
Is it not better to create a Datasource for this migration file instead of reusing a global object?
Please consider that the process in this file runs completely decoupled from the rest of the API.